lib/irb/init.rb

```diff
@@ -289,6 +289,10 @@ def IRB.parse_opts(argv: ::ARGV)
         @CONF[:PROMPT_MODE] = prompt_mode
       when "--noprompt"
         @CONF[:PROMPT_MODE] = :NULL
+      when "--script"
+        noscript = false
```

Comment:
I don't think it's required.  It could be useful if you are building a command line to pass to `irb`, do not know what it contains (may contain `--noscript`), and want to ensure that `irb` treats the first argument as a script.

Reply:
Thanks for explanation 👍 

Yeah let's keep it. But then we need to add it to `help-message` as well.

Reply:
OK, I added `--script` to `help-message`.